package.json

```diff
@@ -10,6 +10,7 @@
 		"build:server": "pnpm nx build @overture-stack/lectern-server",
 		"build:client": "pnpm nx build @overture-stack/lectern-client",
 		"test:all": "pnpm nx run-many --all --target=test",
+		"integration-test:all": "pnpm nx run-many --all --target=integration-test",
```

Comment:
what's the difference between "test" and "integration test"?
I think this change points out that the existing script naming (before this PR) needs some love.

Reply:
the "test" script will run all the test excluding the folder `test/integration/**/*` this exclusion is defined on the default .mocharc.json.

the "integration-test" scripts does the opposite, runs only the tests of the folder `test/integration/**/*`.  (defined on .mocharc.integration.json file